app/retail/templates/settings/email.html

```diff
@@ -41,6 +41,32 @@ <h5>{% trans "Email Preferences" %}</h5>
 
 {% block scripts %}
 <script>
+
+  function changePreferences() {
+    let mauticObj = {
+      email:  document.contxt.email,
+      github_handle: document.contxt.github_handle
+    };
+    let newValues = $('#email-settings input.priv_checkbox').serializeArray()
+    let emailPref = {
```

Comment:
^ const vs let

Reply:
Done

Reply:
emailPref will change values and variables so is the only one now with let